src/stories/AppAutoComplete.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import AppAutoComplete from '~/components/app-auto-complete/AppAutoComplete'
+
+export default {
+  title: 'AppAutoComplete',
+  component: AppAutoComplete,
+  argTypes: {
+    disabled: {
+      type: 'boolean',
+      description: 'Default state of component'
+    },
+    fieldValue: {
+      type: 'string',
+      description: 'Basic value inside field input'
+    },
+    loading: {
+      type: 'boolean',
+      description: 'Loading AutoComplete component'
+    },
+    onChange: {
+      type: 'function',
+      description: 'Function will be called when autocomplete item will be changed'
```

Comment:
Function will be called when autocomplete item _is_ changed

Reply:
fixed